src/components/clusterDetail/ClusterProgress.tsx

```diff
@@ -67,8 +67,16 @@ const ClusterProgress: React.FC<ClusterProgressProps> = ({ cluster }) => {
   return (
     <>
       <DetailList>
-        <DetailItem title="Started on" value={getHumanizedDateTime(cluster.installStartedAt)} />
-        <DetailItem title="Status" value={getInstallationStatus(cluster)} />
+        <DetailItem
+          title="Started on"
+          value={getHumanizedDateTime(cluster.installStartedAt)}
+          idPrefix="cluster-progress-started-on"
```

Comment:
Nit: we could simply call it `id`

Reply:
But we are further appending it so I thought this is clear that such ID will not appear in the DOM